frontend/src/__tests__/cypress/cypress/tests/mocked/home/homeAIFlows.cy.ts

```diff
@@ -103,4 +103,13 @@ describe('Home page AI Flows', () => {
     homeAISection.getTrainFlowCard().find().click();
     homeAISection.findPipelinesTrainDescriptionText().should('not.exist');
   });
+
+  it('should hide the models card when model registry and model serving platform components are disabled regardless of feature flags', () => {
+    homePage.initHomeIntercepts({
+      disableModelRegistry: true,
+      disableModelServing: true,
+    });
```

Comment:
We want to test the inverse case here - leave these features on, and instead disable the backend components. That's what was failing before. You can remove the `initHomeIntercepts` call here and instead call:
```ts
  cy.interceptOdh(
    'GET /api/dsc/status',
    mockDscStatus({
      installedComponents: {
        kserve: false,
        'model-mesh': false,
        'model-registry-operator': false,
      },
    }),
  );
```

Reply:
Done